spec/commands/thredded/moderate_post_spec.rb

```diff
@@ -38,5 +38,27 @@ module Thredded
       expect { Thredded::ModeratePost.run!(post: topic.first_post, moderation_state: :blocked, moderator: moderator) }
         .to_not change { topic.first_post.reload.updated_at }
     end
+
+    context 'when not Thredded.content_visible_while_pending_moderation' do
+      around { |ex| with_thredded_setting(:content_visible_while_pending_moderation, false, &ex) }
+
+      it 'notifies followers on approval' do
```

Comment:
Yeah that's something the hound team has discussed - if/when there's overlap between reek and rubocop. I'll turn it off for this repo for the time being and will use this PR as an example for the group over there.